### PR TITLE
2.1.2 release PR

### PR DESCRIPTION
**Pull request by daschuer**
These should be the commits towards a release.
Is anything missing?

### CODE REVIEW

**Comment by Be-ing** [comment]
#1768 is missing. If 2.1.2 is released without that, I will not spend any effort supporting users with any resultant bugs in 2.1.2.

**Comment by Pegasus-RPG** [comment]
@Be-ing That's fine. Daniel and I will worry about any such bugs. @daschuer Have we actually made sure new strings were pushed to Transifex? I.e. have any translations actually been changed since 2.1.0?

**Comment by daschuer** [comment]
I have only fetched strings from Transiflex. 

You are right, that we also have to push the strings. However this will only have an effect for 2.1.3 if we release this as it is.

**Comment by uklotzde** [comment]
The change set of this PR looks suspicious. It reverts the recent iTunes import fixes!?

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,20 @@
+==== 2.1.2 2018-08-10 ====
+Yet an other bugfix release of Mixxx 2.1.
+Here is a quick summary of what is new in Mixxx 2.1.2:
+
+* Allow maximum deck speed of 4x normal
+* Don't allways quantize hot ques, a 2.1.1 regression lp1777429
```
Comment (Be-ing) [inline CHANGELOG]: two typos: "Don't always quantize hotcues"

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+* Fix artifacts using more than 32 samplers lp1779559
+* store No EQ and Filter persistantly lp1780479
```
Comment (Be-ing) [inline CHANGELOG]: persistently

**Review by Be-ing — CHANGES_REQUESTED** [review]

**Comment by Pegasus-RPG** [comment]
Every time we do a release of any sort (major, minor, bugfix, beta, alpha) we need to push the strings.

**Comment by sblaisot** [comment]
You also need to update windows installer translations:
https://github.com/mixxxdj/mixxx/blob/master/build/wix/Localization/README-Translations.md

**Comment by sblaisot** [comment]
ok, I see there is no new translation for windows installer, so nothing to update.

@daschuer can you add windows installer translation update to the Todo list for the next time please?

**Comment by daschuer** [comment]
Typos fixed, ready for merge and release?

**Comment by sblaisot** [comment]
> Do we have a guide how to push and pull installer translations?

You have the link 2 comments above

**Comment by daschuer** [comment]
@Be-ing can we merge this now?

**Comment by Be-ing** [comment]
I think it's a bad idea without #1768, but okay.

**Comment by daschuer** [comment]
Thank you.

**Pull request merged**